### PR TITLE
Lumicall

### PR DESCRIPTION
**Pull request by MilanKral**

### CODE REVIEW

File: src/chrome/content/rules/Lumicall.org.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	Nonfunctional hosts in *.lumicall.org:
+
+	certificate mismatch:
+	- lists.lumicall.org
+-->
+<ruleset name="Lumicall.org">
+	<target host="lumicall.org" />
+	<target host="www.lumicall.org" />
+
+	<rule from="^http://www\.lumicall\.org/"
```
Comment (koops76) [inline src/chrome/content/rules/Lumicall.org.xml]: Does the ruleset work without this rule?

File: src/chrome/content/rules/OpenTelecoms.org.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	- planet.opentelecoms.org
+	connection refused:
+	- polycom.opentelecoms.org
+	- www.polycom.opentelecoms.org
+-->
+<ruleset name="OpenTelecoms.org">
+	<target host="opentelecoms.org" />
+	<target host="www.opentelecoms.org" />
+
+	<rule from="^http://www\.opentelecoms\.org/"
```
Comment (koops76) [inline src/chrome/content/rules/OpenTelecoms.org.xml]: Does the ruleset work without this rule?

**Comment by MilanKral** [comment]
The Pull Request name was changed.
Please send me a message, if you have any other suggestions.

**Comment by koops76** [comment]
ping @J0WI.

**Pull request merged**